src/webportal/src/app/job/job-view/fabric/job-detail/components/task-role-container-list.jsx

```diff
@@ -130,6 +134,29 @@ PortTooltipContent.propTypes = {
   ports: PropTypes.object,
 };
 
+const LogDialogContent = ({ urlLists }) => {
+  const lists = [];
+  for (const p of urlLists) {
+    lists.push(p);
+  }
+  const urlpairs = lists.map(lists => (
+    <dl key='log-list'>
```

Comment:
Could you use `<Stack></Stack>` instead of `<dl>`? I think it is better in our context.